lua/ttt2/libraries/database.lua

```diff
@@ -393,310 +411,165 @@ end
 -- To be able to directly see message sending- and receive-structure between server and client
 -- They are always paired by either client-Send and server-Receive or server-Send and client-Receive
 
--- Client send and receive functions
-local clientSendFunctions = {}
+-- Client and server receive functions
 local clientReceiveFunctions = {}
-
--- Server send and receive functions
-local serverSendFunctions = {}
 local serverReceiveFunctions = {}
 
 ---
--- Send query for registered databases to server
--- @param table data contains only the messageIdentifier here
+-- An Identifier was received, checks if all sent identifiers were received and resets the identifiers
+-- Should make sure, that it doesnt go to infinity
 -- @realm client
 -- @internal
-clientSendFunctions[MESSAGE_REGISTER] = function(data)
-	net.WriteUInt(data, uIntBits)
+local function receivedIdentifier()
+	identifiersReceived = identifiersReceived + 1
+
+	if identifiersReceived < messageIdentifier then return end
+
+	identifiersReceived = 0
+	messageIdentifier = 0
 end
 
 ---
 -- Receive query for registered databases from client
 -- and sync them
+-- @param table data = {identifier}
```

Comment:
I'm not sure what this doc string means? The description text is `= {identifier}`. That doesn't seem helpful

Reply:
But yeah, could make sense to describe single entries as well

Reply:
do we really need a table here for a single parameter? This makes it hard to understand what type `identifier` is expected by the function for no real gain?

Reply:
Not for this particular function, but for the general workflow, we do. All receiveFunctions follow the same signature of (table, ID64)
And because some functions only need a few parameters, I documented them. In this case its just one 😄